core/workspace_comment_svg.js

```diff
@@ -173,9 +177,6 @@ WorkspaceCommentSvg.prototype.dispose = function() {
   }
 
   dom.removeNode(this.svgGroup_);
-  // Sever JavaScript to DOM connections.
-  this.svgGroup_ = null;
-  this.svgRect_ = null;
```

Comment:
Why remove these?

Reply:
These sets are not necessary and cause compile issues because the type for these two properties is non-nullable (`!`).

Background on why these sets is not necessary is here: https://github.com/google/blockly/issues/2152